libc/functions/u/utimes.part-impl.md

```diff
@@ -1,5 +1,7 @@
 <!-- Documentation template to fill -->
-# Synopsis
+# utimes
```

Comment:
There is no link to this function description in our table of contents - please add it in a separate commit within this PR